style.css

```diff
@@ -338,3 +356,111 @@ textarea {
 .get-in-touch:active {
   background-color: #f3f3f3;
 }
+
+.d-none {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  .main-container {
+    background-image: url('./images/headerbg.svg');
+    align-items: center;
+    justify-content: center;
+    height: 93vh;
+  }
+
+  .header-container {
+    display: flex;
+    align-items: flex-start;
+    justify-content: center;
+    flex-direction: column;
+    max-width: 50rem;
+  }
+
+  .ham-menu {
+    display: none;
+  }
+
+  .d-none {
+    display: block;
+  }
+
+  .d-none-flex {
+    display: flex;
+  }
+
+  .d-none-flex li {
+    margin: 0 20px;
+  }
+
+  .d-none-flex li a:hover {
+    color: var(--font-main-color);
+  }
+
+  .desk-d-grid {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    max-width: 80%;
+    width: auto;
+    padding: 24px;
+    align-items: center;
+    justify-content: center;
+    box-shadow: var(--font-fifth-color) 0 0 10px;
+  }
+
+  .order-2 {
+    order: 2;
+  }
+
+  .image-card img {
+    /* align-items: center;
+    justify-content: center; */
+    text-align: center;
+    width: 100%;
+    height: 100%;
+  }
+
+  .about-myself-section {
```

Comment:
- [ ] The about section has a few details that are not matched.
- Align the left column with the right column from the top.
- The languages items should be aligned in a row and not a column
![Screenshot from 2021-08-19 11-57-13](https://user-images.githubusercontent.com/65501535/130040116-4eb22eb8-3b40-4ab3-a7ed-25929022865d.png)